src/main/scala/com/kinja/soy/SoyWrites.scala

```diff
@@ -75,6 +75,13 @@ trait DefaultSoyWrites {
   }
 
   /**
+   * Converter for Float types.
+   */
+  implicit object FloatSoy extends SoyWrites[Float] {
```

Comment:
Yes.

I was the one who added that `Float` and there wasn't any good reason not to use a `Double` there, I just chose poorly.